frontend/src/components/application/AppVendingControls/SetupControls.tsx

```diff
@@ -32,6 +33,9 @@ interface Props {
 const SetupControls: FunctionComponent<Props> = ({ app, vendingConfig }) => {
   const { t } = useTranslation()
 
+  const [vendingEnabled, setVendingEnabled] = useState(false)
```

Comment:
Does the state combination of vending disabled and still requiring a payment make sense? Just a minor nitpick, as I'm curious @danielsilverstone-ct

Reply:
I can answer this :smiley: 

It doesn't, vending disabled (recommended payment of $0) is the master switch which will entirely clear out the setup when submit to the backend. Also as a rule minimum payment cannot be greater than recommended payment which that kind of setup would imply.

Reply:
I would then go for a custom type instead of two states. Might look into this in some days.